content/4_Gold/Combinatorics.mdx

```diff
@@ -236,15 +266,13 @@ ll fac[MAXN + 1];
 ll inv[MAXN + 1];
 
 //BeginCodeSnip{Counting functions}
-ll exp(ll x, ll y, ll p) {
-	ll res = 1; x %= p;
-	while (y) {
-		if (y & 1) {
-			res *= x; res %= p;
-		}
-		x *= x;
-		x %= p;
-		y >>= 1;
+ll binpow(ll x, ll n, ll m) {
```

Comment:
just curious, why `binpow` instead of `exp`?

Reply:
u refered to this implementation in the last pr as 'the' standard so.., i personally also prefer exp

Reply:
you can replace all `binpow`s with `exp`, i just want consistency really
sorry if i failed to communicate properly